.circleci/get_data.sh

```diff
@@ -1,26 +1,36 @@
 
-if [[ "$SHELL" =~ zsh ]]; then
+if [[ "$SHELL" == zsh ]]; then
   setopt SH_WORD_SPLIT
 fi
```

Comment:
I think `setopt` is a zsh-specific command.

Reply:
it is - this only runs if the shell is zsh though

Reply:
Really? Does `!=` mean that it only runs if it _isn't_ zsh? I keep getting `./get_data.sh: line 3: setopt: command not found` when running with bash.

Reply:
AH I misread the text above. Yes we only want this if it's being run in zsh!